src/Header/Header.css

```diff
@@ -1,6 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredericka+the+Great&display=swap');
+
 header {
   display: flex;
   justify-content: space-around;
   align-items: center;
-  background: radial-gradient(circle at 100%, rgb(131, 19, 19), #333 50%, #eee 75%, #333 75%); 
+	background: linear-gradient(135deg, rgb(131, 19, 19) 60%, #333); 
+	font-family: 'Fredericka the Great';
```

Comment:
I'm all about this font, but I think it could be EVEN BIGGER

Reply:
I'm down to make it bigger! We can def do that.